tensorflow_recommenders_addons/dynamic_embedding/core/kernels/cuckoo_hashtable_op_gpu.cu.cc

```diff
@@ -483,15 +513,21 @@ REGISTER_KERNEL_BUILDER(
     HashTableImportGpuOp);
 
 // Register the CuckooHashTableOfTensors op.
-#define REGISTER_KERNEL(key_dtype, value_dtype)                        \
-  REGISTER_KERNEL_BUILDER(                                             \
-      Name(PREFIX_OP_NAME(CuckooHashTableOfTensors))                   \
-          .Device(DEVICE_GPU)                                          \
-          .TypeConstraint<key_dtype>("key_dtype")                      \
-          .TypeConstraint<value_dtype>("value_dtype"),                 \
-      HashTableGpuOp<                                                  \
-          lookup::CuckooHashTableOfTensorsGpu<key_dtype, value_dtype>, \
-          key_dtype, value_dtype>)
+
+#define REGISTER_KERNEL(key_dtype, value_dtype)                            \
+  REGISTER_KERNEL_BUILDER(                                                 \
+      Name(PREFIX_OP_NAME(CuckooHashTableOfTensors))                       \
+          .Device(DEVICE_GPU)                                              \
+          .TypeConstraint<key_dtype>("key_dtype")                          \
+          .TypeConstraint<value_dtype>("value_dtype"),                     \
+      HashTableGpuOp<                                                      \
+          lookup::CuckooHashTableOfTensorsGpu<key_dtype, value_dtype>,     \
+          key_dtype, value_dtype>);                                        \
+  REGISTER_KERNEL_BUILDER(Name(PREFIX_OP_NAME(CuckooHashTableClear))       \
```

Comment:
It's better to add this code block below [`HashTableClearGpuOp` class implement](https://github.com/yuanqingsunny/recommenders-addons/blob/f2fd10dad6f213a41c023e465b9cdea9c0b12e48/tensorflow_recommenders_addons/dynamic_embedding/core/kernels/cuckoo_hashtable_op_gpu.cu.cc#L440).

Reply:
put here to keep same key_dtype and value_dtype with CuckooHashTableOfTensors